src/modules/controllers/task-controller.js

```diff
@@ -0,0 +1,108 @@
+const Task = require('../models/task');
+const { validationString } = require('../../helpers/validation');
+
+const getAllTasks = async (req, res) => {
+  try {
+    const allTasks = await Task.find().sort({ isCheck: 1 });
+    res.status(200).send({ data: allTasks });
+  } catch (error) {
+    res.status(400).send("Task retrieval error");
+  }
+};
+
+const createNewTask = async (req, res) => {
+  try {
+    const { text } = req.body;
+
+    if (!req.body.hasOwnProperty('text')
+      || !validationString(text)
+    ) {
+      throw new Error("Value not received");
```

Comment:
It makes no sense to write the text of the error, it will never appear anyway. Only the error text in "catch" will work

Reply:
in other functions similarly